File: app/runtime/node/runner.js
const { lambda_handler } = require("./lambda_function");

async function execute_lambda(event, context) {
    let response;
    
    try {
        if (typeof lambda_handler !== "function") {
            throw new Error("lambda_handler is not defined properly.");
        }

        const result = await lambda_handler(event, context);

        response = {
            lambdaStatusCode: 200,
            body: result
        };

    } catch (err) {
        response = {
            lambdaStatusCode: 500,
            body: err?.message || String(err)
        };
    }

    console.log(JSON.stringify(response));

    return response;
}

module.exports = { execute_lambda };

File: app/runtime/python/runner.py
from lambda_function import lambda_handler

def execute_lambda(event, context):
    try:
        result = lambda_handler(event, context)
        return {
            "lambdaStatusCode": 200,
            "body": result
        }
    except Exception as e:
        return {
            "lambdaStatusCode": 500,
            "body": str(e)
        }
